nav2_smart_planner/include/nav2_smart_planner/smart_planner.hpp

```diff
@@ -33,16 +33,18 @@
 namespace nav2_smart_planner
 {
 
-class SmartPlanner : public nav2_tasks::ComputePathToPoseTaskServer
+class SmartPlanner : public rclcpp::Node
 {
 public:
   SmartPlanner();
   ~SmartPlanner();
 
   nav2_tasks::TaskStatus execute(
```

Comment:
If this is no longer an implementation of a task, I think the Execute() method should be renamed to be more descriptive as its no longer an abstract implementation

Reply:
Done.